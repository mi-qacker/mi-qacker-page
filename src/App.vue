<script setup lang="ts">
import NavbarPanel from './components/NavbarPanel.vue';
</script>

<template>
	<NavbarPanel />
	<main class="container mx-auto">
		<RouterView />
	</main>
</template>
