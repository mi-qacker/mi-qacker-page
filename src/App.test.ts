import {mount} from '@vue/test-utils';
import App from './App.vue';

describe('App component', () => {
	it('mount component', async () => {
		expect(App).toBeTruthy();
		const wrapper = mount(App);
		expect(wrapper.html()).toMatchSnapshot();
	});

	it(`title display "Hello world!"`, async () => {
		const wrapper = mount(App);
		const h1 = await wrapper.get('h1');
		expect(h1.text()).toContain('Hello world!');
	});
});
