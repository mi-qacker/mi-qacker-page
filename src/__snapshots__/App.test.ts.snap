// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > mount component 1`] = `"<h1>Hello world!</h1>"`;
